src/plot_styles.py

```diff
@@ -1,46 +1,66 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 import os
-import shutil
 from gettext import gettext as _
 from pathlib import Path
 
 from cycler import cycler
 
-from gi.repository import Adw, GLib, Gtk
+from gi.repository import Adw, GLib, Gio, Gtk
 
 from graphs import file_io, graphs, misc, utilities
 
 
+def _styles_in_directory(directory):
```

Comment:
Why the _ in the function name itself? The function is used in get_system_styles and get_user_styles.

Reply:
That is to indicate this function is for this file only. It is a shared element by the functions you mentioned and is just to avoid having duplicate code.